rust/src/lib.rs

```diff
@@ -138,12 +167,16 @@ pub mod http {
         // 5. Send request
         write!(
             stream,
-            "GET {} HTTP/1.0\r
+            "GET {} HTTP/1.1\r
 Host: {}\r
+Connction: close\r
+User-Agent: Mozilla/5.0 ({})\r
 Accept-Encoding: gzip,deflate\r
 \r
 ",
```

Comment:
This is not a decision from this PR, but you may want to use multiline whitespace escape machanism
https://doc.rust-lang.org/rust-by-example/std/str.html#literals-and-escapes